File: go-client-sdk/pkg/models/shared/runprojectrequestbody.go
package shared

// RunProjectRequestBody
// Request body format for triggering a project run
type RunProjectRequestBody struct {
	DryRun      *bool                  `json:"dryRun,omitempty"`
	InputParams map[string]interface{} `json:"inputParams,omitempty"`
	UpdateCache *bool                  `json:"updateCache,omitempty"`
}

File: typescript-client-sdk/src/sdk/models/shared/runprojectrequestbody.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";
import { Expose } from "class-transformer";


// RunProjectRequestBody
/** 
 * Request body format for triggering a project run
**/
export class RunProjectRequestBody extends SpeakeasyBase {
  @SpeakeasyMetadata()
  @Expose({ name: "dryRun" })
  dryRun?: boolean;

  @SpeakeasyMetadata()
  @Expose({ name: "inputParams" })
  inputParams?: Record<string, any>;

  @SpeakeasyMetadata()
  @Expose({ name: "updateCache" })
  updateCache?: boolean;
}